pkgs/applications/misc/survex/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, autoreconfHook
+, pkg-config
+, wxGTK30-gtk3
+, ffmpeg
+, proj
+, perl532
+, unscii
+, python
+, libGL
+, libGLU
+, x11
```

Please don't use aliases in Nixpkgs derivations.

https://github.com/NixOS/nixpkgs/blob/7e529086a7ce588380178733a42ff2a293ac080c/pkgs/top-level/aliases.nix#L875

```suggestion
, xlibsWrapper
```

---

Actually it seems like xlibsWrapper shouldn't be used in general.

https://github.com/NixOS/nixpkgs/blob/15c093a16789a66befd3acb56a6bc18441b7b2b8/pkgs/top-level/all-packages.nix#L18617-L18618

Could you remove it and manually add whatever X11-relevant libraries it asks for?